mix.exs

```diff
@@ -19,12 +20,17 @@ defmodule Klife.MixProject do
     ]
   end
 
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:klife_protocol, "~> 0.1.0"},
       # Benchmarks and tests
-      {:benchee, "~> 1.0", only: :dev, runtime: false}
+      {:benchee, "~> 1.0", only: :dev, runtime: false},
+      # Used to wait for connection in tests
+      {:retry, "~> 0.18", only: :test}
```

Comment:
As I've mentioned on the other comment, we may not need this dependency for this simple use case.